content-sharing/Yasas.json

```diff
@@ -0,0 +1,6 @@
+{
+    "favourite-game": "yasas2002",
+    "favourite-language": "Java",
+    "favourite-color": "blue"
+    "Projects" : ["https://friendly-lewin-772370.netlify.app/"],
+  }
```

Comment:
Follow the exact same JSON file template provided in the readme file.
Your pull request does not align with the JSON syntax provided. 
Please follow the exact template as mentioned below
```
{
    "github-username": "YOUR-GITHUB-USERNAME",
    "favourite-game": "YOUR-FAVOURITE-GAME",
    "favourite-language": "YOUR-FAVOURITE-PROGRAMMING-LANGUAGE",
    "favourite-color": "#FFFFFF"
}
```